docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -64,10 +54,33 @@ otherwise the {classanalysis} may not provide the best predictions. Read
 The ensemble algorithm that we use in the {stack} for {classification} is a type 
 of boosting called boosted tree regression model which combines multiple weak 
 models into a composite one. We use decision trees to learn to predict the 
-probability that a data point belongs to a certain class.
+probability that a data point belongs to a certain class. A sequence of decision 
+trees are trained and every decision tree learns from the mistakes of the 
+previous one. Every tree is an iteration of the last one, hence it improves the 
+decision made by the previous tree.
 //end::classification-algorithms[]
 
 
+[discrete]
+[[dfa-classification-performance]]
+==== {classification-cap} performance
+
+As a rule of thumb, a {classanalysis} with many classes takes more time to run 
+than a binary {classification} process when there are only two classes. The 
+relationship between the number of classes and the runtime is linear.
+
+The runtime also scales approximately linearly with the number of involved 
+documents below 200.000 data points. Therefore, if you double the number of 
+documents, then the runtime of the analysis doubles respectively. Above 200.000 
+data points, runtime increases faster.
+
+To improve the performance of your {classanalysis}, consider to set a smaller 
```

Comment:
> To improve the performance of your {classanalysis}, consider to set a smaller training percent. 

When I read this, it's not immediately obvious what this means.  I had to go look at the API for a definition. Could we maybe change it to something like this?:

> To improve... consider using a smaller `training_percent` value when you create the job. That is to say, use a smaller percentage of your documents to train the model more quickly.